docs/workflows/sdk-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+title: SDK Setup
+sidebar_position: 8
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+The Embedded SDK is a holistic SDK solution offering the entire experience embedded in your product. Users will not need to download the Beyond Identity Authenticator. A set of functions are provided to you through the `Embedded` namespace. This SDK supports OIDC and OAuth2.
+
+## Sample App
+
+<Tabs groupId="sdks">
+<TabItem value="android" label="Android">
```

Comment:
I wonder if we should agree on an order for displaying the SDKs and do that for all the 1-pagers.

Maybe we can either display them like in the console:
JS, Kotlin, Swift, React Native, Flutter
or 
alphabetically. 

I don't really have a preference as long as they are consistent.

Reply:
I generally like alphabetically, though I'm open. The SDKs section is currently ordered as Javascript SDK, Kotlin SDK, Swift SDK, Flutter SDK, React Native SDK. Maybe we maintain that order?

Reply:
I tend to like alphabetically too, unless there's a good reason not to do that. I'm not sure if we have an order preference for some marketing reason? @ismailmustafa @nmelo do you know?

Reply:
I think we should go with the order determined in console, just to keep things consistent.